charts/turing/templates/turing-cluster-role-binding.yaml

```diff
@@ -0,0 +1,16 @@
+{{- if .Values.clusterConfig.useInClusterConfig -}}
```

Comment:
Is it necessary for all files in `templates` directory to have `turing` prefix?

Reply:
Not really, but that's the convention the Merlin templates are following. So I did the same.